source/system/external/windows.ooc

```diff
@@ -0,0 +1,105 @@
+/* This file is part of magic-sdk, an sdk for the open source programming language magic.
+ *
+ * Copyright (C) 2016 magic-lang
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+version(windows) {
```

Comment:
Code in this file is identical to before, I just rearranged some stuff to get variables and covers/classes at the top, and removed indentation.